OpenUpSA/wazimap-ng-ui
# Feat: Data Download for new data structure

## Description
making data download work with the new data structure

## Related Issue


## How to test it locally
* expand the rich data panel
* using the popup menu of any chart download data as CSV, Excel and JSON and confirm that the downloaded data is correct.

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/115222020-f2cdf800-a112-11eb-95e5-7b2a1ffe9e26.png)


## Changelog

### Added

### Updated
* `chart.js` to add / modify the functions that will be used to download data

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
